backend/src/api/transaction-utils.ts

```diff
@@ -116,7 +116,7 @@ class TransactionUtils {
   public extendMempoolTransaction(transaction: IEsploraApi.Transaction): MempoolTransactionExtended {
```

Comment:
The Sigop data needs to be added to `extendTransaction` function to be shown on Transaction pages in Bitcoin Core only mode. Is it missing or does it have some issues with it?